.rubocop_todo.yml

```diff
@@ -32,7 +32,13 @@ RSpec/AnyInstance:
     - 'spec/rubocop/cop/lint/duplicate_methods_spec.rb'
     - 'spec/rubocop/target_finder_spec.rb'
 
-# Offense count: 2574
+# Offense count: 1004
+# Configuration parameters: Prefixes.
+# Prefixes: when, with
+RSpec/ContextWording:
```

Comment:
Is it better to wrote .rubocop.yml?

Reply:
It's okay to keep it here. We have a lot of legacy in our specs, but at some point I hope we'll catch up. 🙂